packages/adminrouter/extra/src/includes/http/open/master.conf

```diff
@@ -1,3 +1,10 @@
+# Enable VTS module for metrics
+# Set the shared memory zone size to 256MB to be sure to
```

Comment:
Let's be more precise with why 256 was chosen, and how someone wanting to change this number can know if their choices are sensible.

Does this mean that we have increased the memory requirement for DC/OS by 256 MB / master + 32 MB / agent?

Reply:
Yes, that's what it means (I believe).